src/level/level.cpp

```diff
@@ -15,13 +16,18 @@ namespace level {
 
         segments_ = datastruct::Vector<WallSegment *>();
         entities_ = datastruct::Vector<Entity *>(0);
+        artifacts_ = datastruct::Vector<Artifact *>();
+
         // creazione dei muri esterni
         using namespace enums; // Direction::RIGHT, Direction::LEFT, Direction::UP, Direction::DOWN
         segments_.push_back(new WallSegment(Position(0, 0), Direction::RIGHT, size.colonna - 1));
         segments_.push_back(new WallSegment(Position(0, size.colonna - 1), Direction::DOWN, size.riga - 1));
         segments_.push_back(new WallSegment(Position(size.riga - 1, size.colonna - 1), Direction::LEFT, size.colonna - 1));
         segments_.push_back(new WallSegment(Position(size.riga - 1, 0), Direction::UP, size.riga - 1));
 
+        //inserimento in mappa degli artefatti
+        artifacts_.push_back(new Artifact(1, Position(10,5)));
```

Comment:
questa cosa andrà in maledetto conflitto con la pr di angi

Reply:
si, si dovrà fare un rebase, e risolverli nel momento del rebase, comunque se mergiamo prima questa sarà più facile perché segue un pò l'ordine storico. 

Ora dovrebbe Simo però fare un rebase già sul master, perché questo andava in conflitto con il door che ho già mergiato.